### DISCUSSION TITLE
MQTT ohne Verschlüsselung sicher?

### DISCUSSION THREAD

**Discussion by Arnie-75** [tbnobody/OpenDTU#1490] [General]
Hallo,
ich schicke die Daten des OpenDTU über Port 8883 mit TLS per MQTT an mqtt.eclipseprojects.io . Ohne Benutzername und Passwort. Das Häkchen darunter für TLS Login habe ich nicht aktiviert. 
Dass die verschickten Daten des OpenDTU von jedem lesbar sind ist mir klar. Aber ist meine OpenDTU dadurch von außen auch potentiell angreifbar, oder ist MQTT eine Einbahnstraße?

**Comment by DPO99** [comment]
Wieso von außen? Du solltest deine DTU nicht aus dem Internet erreichbar machen. So einfach ist das. Wenn du von außen in dein Netz willst, nutze VPN.

**Comment by Arnie-75** [comment]
Danke für die Antwort. Meine DTU ist grundsätzlich nicht von außen erreichbar, es sei denn das Verschicken per MQTT an einen MQTT-Server im Internet sorgt genau dafür. Das wäre meine Frage.

**Comment by DPO99** [comment]
Hm also ein Problem hat ja eher der nicht geschützte MQTT Server. Wenn du von OpenDTU wirklich nur auf diesen gehst, sollte das kein Problem sein. Nichts desto trotz würde ich TLS mindestens aktivieren und wie gesagt, kein Port-Forwarding oder sowas für die DTU einrichten. Von Extern nur über VPN erreichbar machen.

Die Kernfrage die sich mir aber stellt ist: Wieso betreibst du nicht deinen eigenen MQTT Server? Wieso raus ins Internet? Das würde ich mit solchen Geräten generell niemals machen.

**Comment by Arnie-75** [comment]
Danke, DTU ist von außen nur per VPN erreichbar.
Ob ich einen eigenen MQTT Server baue weiß ich noch nicht. Möchte eigentlich keine neue Hardware installieren, aber ich denke mal darüber nach.
